readme.md

```diff
@@ -1 +1,67 @@
+# Gym Landing Page
```

Comment:
I am not sure why do you create new `readme.md`?

Reply:
- [ ] Delete `readme.md` from your changes. We already have `README.md`.

Reply:
@Min-Ho-Lim ok